app/controllers/notification/NotificationCallbackController.scala

```diff
@@ -16,20 +16,35 @@
 
 package controllers.notification
 
+import config.AppConfig
 import javax.inject.{Inject, Singleton}
 import play.api.Logger
 import play.api.mvc.Action
 import services.NotificationService
 import uk.gov.hmrc.http.BadRequestException
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 
-import scala.concurrent.ExecutionContext
+import scala.concurrent.{ExecutionContext, Future}
+import scala.xml.NodeSeq
 
 @Singleton
-class NotificationCallbackController @Inject()(notificationService: NotificationService)(implicit ec: ExecutionContext) extends FrontendController {
+class NotificationCallbackController @Inject()(notificationService: NotificationService,implicit val appConfig: AppConfig)(implicit ec: ExecutionContext) extends FrontendController {
```

Comment:
Why is the appConfig implicit?  This is usually done when we need it for a view.  Which we don't here.